addons/survey/static/src/js/survey_choice_image_zoomer.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('survey.choice_image_zoomer', function (require) {
+    'use strict';
+
+var publicWidget = require('web.public.widget');
+
+publicWidget.Widget.surveyChoiceImageZoomer = publicWidget.Widget.extend({
+    template: 'survey.survey_choice_image_zoomer',
+    xmlDependencies: ['/survey/static/src/xml/survey_choice_image_zoomer_templates.xml'],
+    events: {
+        'DOMMouseScroll .o_survey_img_zoomer_image': '_onImgScroll', // Firefox
+        'mousewheel .o_survey_img_zoomer_image': '_onImgScroll', // Chrome, Safari, IE
+        'click': '_onZoomerClick', // Allow user to close by clicking anywhere (mobile...)
+        'click .o_survey_img_zoomer_in_btn': '_onZoomIn',
+        'click .o_survey_img_zoomer_out_btn': '_onZoomOut',
+    },
+    /**
+     * @override
+     */
+    init(params) {
+        this.zoomImageScale = 1;
+        this.zoomStep = 0.1;
+        // The image is needed to render the template survey_choice_image_zoomer.
+        this.sourceImage = params.sourceImage;
+        return this._super(... arguments);
+    },
+    /**
+     * Open a transparent modal displaying the survey choice image.
+     * @override
+     */
+    async start() {
+        let self = this;
+        await this._super(...arguments).then(() => {
```

Comment:
Either you do it "old school" with a .then.

Either you await and remove the "then".

```js
const superResult = await this._super(...arguments);
// your stuff
return superResult;
```